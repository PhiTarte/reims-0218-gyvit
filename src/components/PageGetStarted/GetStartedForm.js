import React from "react"
import { Container } from "reactstrap"
import GetStartedSignInEmail from "./GetStartedSignInEmail"
import GetStartedSignInTitle from "./GetStartedSignInTitle"
import GetStartedSignInButton from "./GetStartedSignInButton"
import GetStartedSignInLinkHaveAnAccount from "./GetStartedSignInLinkHaveAnAccount"

const GetStartedForm = () => (
  <Container
    style={{
      paddingTop: "25%",
      marginLeft: "30px",
      backgroundColor: "#ffffff",
      marginRight: "50px"
    }}
  >
    <GetStartedSignInTitle />
    <GetStartedSignInEmail />
    <GetStartedSignInButton />
    <GetStartedSignInLinkHaveAnAccount />
  </Container>
)

export default GetStartedForm
