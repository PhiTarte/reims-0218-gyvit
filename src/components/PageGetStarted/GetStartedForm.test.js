import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import GetStartedForm from "./GetStartedForm"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<GetStartedForm />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<GetStartedForm />).toJSON()
  expect(tree).toMatchSnapshot()
})
