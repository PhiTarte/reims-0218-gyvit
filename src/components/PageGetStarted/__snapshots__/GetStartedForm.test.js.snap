// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
  style={
    Object {
      "backgroundColor": "#ffffff",
      "marginLeft": "30px",
      "marginRight": "50px",
      "paddingTop": "25%",
    }
  }
>
  <div>
    <p
      style={
        Object {
          "color": "#3b425d",
          "fontSize": "24px",
          "fontStretch": "normal",
          "fontStyle": "normal",
          "fontWeight": "normal",
          "height": "24px",
          "letterSpacing": "normal",
          "lineHeight": "normal",
          "textAlign": "left",
        }
      }
    >
      Get started absolutely free
    </p>
    <p
      style={
        Object {
          "color": "#9da5c9",
          "fontSize": "18px",
          "fontStyle": "normal",
          "fontWeight": "normal",
          "height": "18px",
          "letterSpacing": "normal",
          "textAlign": "left",
          "width": "300px",
        }
      }
    >
      Free forever. No credit card needed.
    </p>
  </div>
  <div
    style={
      Object {
        "marginBottom": "40px",
        "marginTop": "60px",
      }
    }
  >
    <div
      className="form-group"
    >
      <label
        className=""
        htmlFor="email"
        style={
          Object {
            "color": "#9da5c9",
            "fontSize": "12px",
            "fontStyle": "normal",
            "fontWeight": "normal",
            "height": "12px",
            "letterSpacing": "normal",
            "marginTop": "30px",
            "textAlign": "left",
            "width": "33px",
          }
        }
      >
        Email
         
      </label>
      <input
        className="form-control"
        id="IdEmail"
        name="email"
        placeholder="Email"
        type="email"
      />
    </div>
    <div
      className="form-group"
    >
      <label
        className=""
        htmlFor="email"
        style={
          Object {
            "color": "#9da5c9",
            "fontSize": "12px",
            "fontStyle": "normal",
            "fontWeight": "normal",
            "height": "12px",
            "letterSpacing": "normal",
            "marginTop": "30px",
            "textAlign": "left",
            "width": "33px",
          }
        }
      >
        Name
         
      </label>
      <input
        className="form-control"
        id="IdEmail"
        name="name"
        placeholder="Name"
        type="email"
      />
    </div>
    <div
      className="form-group"
    >
      <label
        className=""
        htmlFor="password"
        style={
          Object {
            "color": "#9da5c9",
            "fontSize": "12px",
            "fontStyle": "normal",
            "fontWeight": "normal",
            "height": "12px",
            "letterSpacing": "normal",
            "marginTop": "30px",
            "textAlign": "left",
            "width": "75px",
          }
        }
      >
        Password
      </label>
      <input
        className="form-control"
        id="IdPassword"
        name="password"
        placeholder="Password"
        type="password"
      />
    </div>
  </div>
  <div>
    <button
      className="btn btn-info"
      onClick={[Function]}
      style={
        Object {
          "backgroundImage": "linear-gradient(to left, #57aad1, #7a57d1)",
          "borderRadius": "100px",
          "height": "54px",
          "width": "192px",
        }
      }
      type={undefined}
    >
      Continue
    </button>
  </div>
  <div
    style={
      Object {
        "color": "#3b425d",
        "fontSize": "16px",
        "fontStyle": "normal",
        "fontWeight": "normal",
        "height": "16px",
        "letterSpacing": "normal",
        "marginTop": "120px",
        "textAlign": "left",
        "width": "300px",
      }
    }
  >
    <p>
      Already have an account ?
      <a
        className="nav-link"
        href="#"
        onClick={[Function]}
        style={
          Object {
            "display": "inline",
          }
        }
      >
        Sign in
      </a>
    </p>
  </div>
</div>
`;
